api/src/main/java/org/apache/brooklyn/api/objs/BrooklynObject.java

```diff
@@ -141,4 +149,11 @@
          */
         boolean unsubscribe(SubscriptionHandle handle);
     }
+    
+    public interface RelationSupport<T extends BrooklynObject> {
+        public <U extends BrooklynObject> void add(Relationship<? super T,U> relationship, U target);
```

Comment:
Should we add `? super U` for the `Relationship`, for the same reason we have it for the `T`?